airbyte-cdk/python/bin/generate-component-manifest-files.sh

```diff
@@ -18,7 +18,6 @@ function main() {
     docker run --user "$(id -u):$(id -g)" -v "$ROOT_DIR":/airbyte airbyte/code-generator:dev \
       --input "/airbyte/$YAML_DIR/$filename_wo_ext.yaml" \
       --output "/airbyte/$OUTPUT_DIR/$filename_wo_ext.py" \
-      --use-title-as-name \
```

Comment:
the components that have names in master are
- SessionTokenAuthenticator
- AuthFlow
- DeclarativeSource

None of them are being renamed by removing this flag because the title matched the name